lib/types/cql-utils.js

```diff
@@ -7,17 +7,72 @@ const TimeUuid = require("./time-uuid");
 const Uuid = require("./uuid");
 const Duration = require("./duration");
 
+/**
+ * Ensures the value isn't one of many ways to express null value
+ * @param {string} message Error message if value is "null"
+ * @throws {TypeError} Throws an error if the value is "null"
+ */
+function ensureValue(val, message) {
+    if (val === null || typeof val === "undefined" || val === types.unset)
+        throw new TypeError(message);
+}
+
+/**
+ * Wraps each of the elements into given subtype
+ * @param {Array} values
+ * @param {rust.ComplexType} subtype
+ * @returns {Array<rust.QueryParameterWrapper>}
+ */
+function encodeList(values, subtype) {
+    if (!Array.isArray(values))
+        throw new TypeError(
+            `Not a valid list value, expected Array obtained ${util.inspect(values)}`,
+        );
```

Comment:
:question: I don't understand the syntax of `expected Array obtained [values]`.

Reply:
This was taken directly form datastax code: https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/2a750ff0e11955b04161ab4a05feeda1855a827e/lib/encoder.js#L816-L819
I can update this message to something more meaningful.

Reply:
Well, if it comes from the Datastax driver, then users should probably be already used to it...